tests/wp-cldr-tests.php

```diff
@@ -219,4 +221,77 @@ public function test_first_day_of_week() {
 		$this->assertEquals( '', $this->cldr->first_day_of_week( 'bad-slug' ) );
 		$this->assertEquals( '', $this->cldr->first_day_of_week( '' ) );
 	}
+
+	public function test_get_currency_for_all_countries() {
+
+		$this->assertEquals( 'USD', $this->cldr->get_currency_for_all_countries()['US'] );
```

Comment:
Not sure why, but it looks like this is failing the CI test. Probably some sort of static analysis check.  Maybe pull the `$this->cldr->get_currency_for_all_countries()` result to an intermediate variable and run the assert on it instead.